### PR TITLE
Feature/us8427 clean up hotfix css

### PR DESCRIPTION
**Pull request by tatelucas**

### CODE REVIEW

File: crossroads.net/styles/hotfix-y5.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+.crds-styles .nav-trips .nav >li:not(:last-child)> a:hover {
+    background-color: transparent;
+    color: #fff;
+}
+.crds-styles .nav-trips .nav >li:not(:last-child)> a:focus {
+    background-color: transparent;
+    color: #fff;
+}
+.crds-legacy-styles .nav-trips .navbar-nav>li>a {
+  padding-top: 10px;
```
Comment (lukemoderwell) [inline crossroads.net/styles/hotfix-y5.css]: maybe `padding: 10px 0;` ?

File: crossroads.net/styles/hotfix-y5.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+.crds-styles .nav-trips .navbar-nav>li>a {
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+#goTripsNav .navbar-nav >li> a {
+  color: #fff;
+}
+#goTripsNav .navbar-nav >li:not(:last-child)> a:hover,
```
Comment (lukemoderwell) [inline crossroads.net/styles/hotfix-y5.css]: Could these selectors just be added to line `3`?

File: crossroads.net/styles/hotfix-y5.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  border-color: #0095D9;
+}
+@media only screen and (max-width:768px){
+  #goTripsNav {
+    background-color: #fff;
+  }
+  #goTripsNav .navbar-nav >li:not(:last-child)> a:hover,
+  #goTripsNav .navbar-nav >li:not(:last-child)> a:focus,
+  #goTripsNav .navbar-nav >li:not(:last-child)> a:active {
+    background-color: transparent;
```
Comment (lukemoderwell) [inline crossroads.net/styles/hotfix-y5.css]: Do you need this when you have line `26`?

File: crossroads.net/styles/pages/extend-the-weekend.scss [hunk 4]
```diff
@@ -0,0 +1,77 @@
+/* Extend the Weekend */
+.extend-the-weekend {
+  font-family: "acumin-pro", sans-serif;
```
Comment (lukemoderwell) [inline crossroads.net/styles/pages/extend-the-weekend.scss]: Question: should you throw `Helvetica` in there after acumin?

File: crossroads.net/styles/pages/extend-the-weekend.scss [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    font-weight: 300;
+  }
+
+  .intro {
+    h1 {
+      font-size: 238%;
+      line-height: 100%;
+
+      @media (min-width: $screen-sm) {
+        font-size: 338%;
```
Comment (lukemoderwell) [inline crossroads.net/styles/pages/extend-the-weekend.scss]: ??

Comment (tatelucas) [inline crossroads.net/styles/pages/extend-the-weekend.scss]: Removed

File: crossroads.net/styles/pages/extend-the-weekend.scss (hunk 4 above)
Comment (tatelucas) [inline crossroads.net/styles/pages/extend-the-weekend.scss]: I just replicated what we had in other parts of the site when using Acumin.

File: crossroads.net/styles/hotfix-y5.css (hunk 2 above)
Comment (tatelucas) [inline crossroads.net/styles/hotfix-y5.css]: Good call.

File: crossroads.net/styles/hotfix-y5.css (hunk 3 above)
Comment (tatelucas) [inline crossroads.net/styles/hotfix-y5.css]: 👍

**Pull request merged**